phoebe/backend/universe.py

```diff
@@ -3326,3 +3331,260 @@ def process_teffs(self, teffs, coords, s=np.array([0., 0., 1.]), t=None):
             return teffs
 
         raise NotImplementedError("teffext=True not yet supported for pulsations")
+
```

Comment:
I would remove all this.

Reply:
I did not touch this file. Must have appeared after the rebase. @kecnry can you advise?

Reply:
looks like this was an attempt to use a feature to do the smoothing - definitely something we _might_ want to reconsider, but if its commented out, let's remove it